packages/create-gasket-app/test/unit/scaffold/actions/generate-files.test.js

```diff
@@ -1,42 +1,105 @@
-import { jest } from '@jest/globals';
 import { fileURLToPath } from 'url';
 import fs from 'fs';
 import path from 'path';
-const mockReadFileStub = jest.fn();
-const mockWriteFileStub = jest.fn();
-const mockMkdirpStub = jest.fn();
-const mockGlobStub = jest.fn();
+const mockReadFileStub = vi.fn();
+const mockWriteFileStub = vi.fn();
+const mockMkdirpStub = vi.fn();
+const mockGlobStub = vi.fn();
 let registerHelperSpy;
 
-jest.unstable_mockModule('fs/promises', () => {
+vi.mock('fs/promises', () => {
   return {
     default: {
       readFile: mockReadFileStub,
       writeFile: mockWriteFileStub
     }
   };
 });
-jest.mock('mkdirp', () => mockMkdirpStub);
-jest.mock('handlebars', () => {
+vi.mock('mkdirp', () => ({ default: mockMkdirpStub }));
+vi.mock('handlebars', () => {
   return {
-    create: () => {
-      const mod = jest.requireActual('handlebars');
-      const handlebars = mod.create();
-      registerHelperSpy = jest.spyOn(handlebars, 'registerHelper');
-      return handlebars;
+    default: {
+      create: () => {
```

Comment:
This mock seems very heavy. Can you try using's vitest's equivalent  `vi.importActual`.

You may have to await import outside the mock first. Untested, but something like:

```js
const actualHandlebars = await vi.importActual('handlebars');

jest.mock('handlebars', () => {
  return {
    create: () => {
      const handlebars = actualHandlebars.create();
      registerHelperSpy = jest.spyOn(handlebars, 'registerHelper');
      return handlebars;
    }
  };
});
```